src/components/ChatApp/MessageComposer.react.js

```diff
@@ -22,7 +22,7 @@ const loadingStyle = {
     fill: '#607d8b',
     marginTop: '10px'
 }
-let Button, speechRecog;
+let Button = <Mic />, speechRecog;
```

Comment:
We cannot have the Button to be `<Mic />` by default as it first needs to be checked whether the Browser supports the Speech API. So it is just initialized that way. Also, I am not able to produce this issue in http://chat.susi.ai Can you please check once? @madhavrathi